app/views/restaurants/new.html.haml

```diff
@@ -3,30 +3,4 @@
     - @restaurant.errors.full_messages.each do |message|
```

Comment:
Nevermind this comment @MikaelFeher It was based on the image I saw in the PR's description which has now been updated accordingly. I'll submit a PR later today to fix the flash messages